recipe/meta.yaml

```diff
@@ -22,6 +22,8 @@ requirements:
     - pip
     - python
     - pdm-pep517 >=0.3.0
```

Comment:
Should be >=1.0
https://github.com/pdm-project/pdm/blob/2.2.1/pyproject.toml#L167

Reply:
I meant in the host section.
The host section should have:
```
host:
  - pdm-pep517 >=1.0
```
(https://github.com/pdm-project/pdm/blob/2.2.1/pyproject.toml#L167)
 And the run section should still have:
```
 run:
  - pdm-pep517 <2.0.0,>=1.0.0
```
(https://github.com/pdm-project/pdm/blob/2.2.1/pyproject.toml#L25)

Reply:
Fixed this. Thanks!